authorizerd.go

```diff
@@ -453,6 +453,10 @@ func (a *authority) Authorize(r *http.Request, act, res string) (Principal, erro
 
 // VerifyRoleCert verifies the role certificate for specific resource and return and verification error.
 func (a *authority) VerifyRoleCert(ctx context.Context, peerCerts []*x509.Certificate, act, res string) error {
+	if a.disablePolicyd {
```

Comment:
Why don't you surround CheckPolicy with if?
access toekn and roletoken are surrounded by CheckPolicy with if(L417-L422).

Reply:
Fixed
https://github.com/yahoojapan/athenz-authorizer/pull/76/commits/9b6ee66ed2c6f2d483eb364d58091468538b97a2